src/charts/donut.js

```diff
@@ -225,9 +229,8 @@ define(function(require) {
                 d.quantity = +d[quantityLabel];
```

Comment:
could we use object spread here too!? I remember running into some build issues when i tried a while back, maybe its fixed now